rdsbroker/broker.go

```diff
@@ -853,7 +865,18 @@ func (b *RDSBroker) createDBInstance(instanceID string, servicePlan ServicePlan,
 
 	parameterGroupName, err := b.parameterGroupsSelector.SelectParameterGroup(servicePlan, provisionParameters)
 	if err != nil {
-		return &rds.CreateDBInstanceInput{}, err
+		return nil, err
+	}
+
+	tags := RDSInstanceTags{
+		Action:                   "Created",
+		ServiceID:                details.ServiceID,
+		PlanID:                   details.PlanID,
+		OrganizationID:           details.OrganizationGUID,
+		SpaceID:                  details.SpaceGUID,
+		SkipFinalSnapshot:        skipFinalSnapshotStr,
+		OriginSnapshotIdentifier: "",
```

Comment:
This line is unnecessary, but I don't mind if it stays here.

Reply:
Oops, yeah, missed that one! I'll tidy it up.